docs/user/openstack/customization.md

```diff
@@ -234,6 +235,32 @@ controlPlane:
 
 **NOTE:** The additional security groups attached to the Control Plane machine will also be attached to the bootstrap node.
 
+## Cloud Provider configuration
+
+You may want to modify cloud provider configuration in order to make it work with your OpenStack cloud. This is possible if you'll let the installer generate the manifests before running the installation:
+
+```sh
+openshift-install --dir <directory> create manifests
+```
+
+Then modify the manifest containing the cloud provider configuration:
```

Comment:
Should we perhaps point to https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf for all available cloud provider options?

Reply:
Depends on if we want to advertise options that we don't support or would simply not work because legacy cloud provider is somehow broken with newer Octavia?

Reply:
I still believe we should point to the cloud provider cloud.conf page. We may chose to be more prescriptive in the product doc, downstream.

Reply:
Ok.